index.html

```diff
@@ -20,35 +20,38 @@
 <body data-theme="theme1">
 
   <nav class="top">
-    <a id="toplink" href="#footerlink">Ugrás a lap aljára</a>
+    <a id="toplink" href="#footerlink" aria-label="Jump to the bottom of the page">Ugrás a lap aljára</a>
   </nav>
 
-  <div class="base-image">
+  <div class="base-image" role="figure">
     <img src="https://picsum.photos/2200/400" alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit"
-         loading="lazy">
+         loading="lazy" >
   </div>
   <nav class="sublinks">
-    <a href="#subsipsum1">Lorem subIpsum 1</a>
-    <a href="#subsipsum2">Lorem subIpsum 2</a>
-    <a href="#subsipsum3">Lorem subIpsum 3</a>
-    <a href="#subsipsum4">Lorem subIpsum 4</a>
-    <a href="#subsipsum5" target="_blank">Lorem subIpsum 5</a>
-    <a href="#subsipsum6" target="_blank">Lorem subIpsum 6</a>
+    <a href="#subsipsum1" aria-label="Jump to article paragraph" >Lorem subIpsum 1</a>
+    <a href="#subsipsum2" aria-label="Jump to article paragraph" >Lorem subIpsum 2</a>
+    <a href="#subsipsum3" aria-label="Jump to article paragraph" >Lorem subIpsum 3</a>
+    <a href="#subsipsum4" aria-label="Jump to article paragraph" >Lorem subIpsum 4</a>
+    <a href="#subsipsum5" aria-label="Jump to article paragraph" target="_blank">Lorem subIpsum 5</a>
+    <a href="#subsipsum6" aria-label="Jump to article paragraph" target="_blank">Lorem subIpsum 6</a>
   </nav>
 
   <div class="theme-selector-box">
-    <button id="theme1" type="button" class="btn btn-secondary theme-selector" data-theme="theme1">Theme 1</button>
-    <button id="theme2" type="button" class="btn btn-secondary theme-selector" data-theme="theme2">Theme 2</button>
-    <input type="text" id="article-search" placeholder="Search in article">
+    <button id="theme1" type="button" class="btn btn-secondary theme-selector" data-theme="theme1"
+            aria-label="Theme change"  >Theme 1</button>
```

Comment:
Melyik temara fog valtani ez a gomb, az 1-esre vagy a 2-esre? A felolvaso program csak annyit fog mondani erre a  gombra es a következöre is, hogy "Theme change".